ui-v2/app/templates/dc/services/instance/proxy.hbs

```diff
@@ -0,0 +1,96 @@
+<div class="tab-section">
+  <div role="tabpanel">
+  {{#if (or (gt proxy.ServiceChecks.length 0) (gt proxy.NodeChecks.length 0))}}
+    <section>
+      <h3>Proxy health checks</h3>
+      <HealthcheckList data-test-proxy-checks @items={{append proxy.ServiceChecks proxy.NodeChecks}} />
+    </section>
+  {{/if}}
+  {{#if (gt proxy.Proxy.Upstreams.length 0)}}
+    <section class="proxy-upstreams">
+      <h3>Upstreams</h3>
+      <ul data-test-proxy-upstreams>
+      {{#let proxy.Datacenter as |proxyDatacenter|}}
+        {{#each proxy.Proxy.Upstreams as |item|}}
+          <li>
+            <p data-test-destination-name>
+              {{item.DestinationName}}
+            </p>
+            <ul>
+          {{#if (env 'CONSUL_NSPACES_ENABLED')}}
+            {{#if (not-eq item.DestinationType 'prepared_query')}}
+              <li class="nspace">
+                {{or item.DestinationNamespace 'default'}}
+              </li>
+            {{/if}}
+          {{/if}}
+            {{#if (and (not-eq item.Datacenter proxyDatacenter) (not-eq item.Datacenter ""))}}
+              <li class="datacenter">
+                {{item.Datacenter}}
+              </li>
+            {{/if}}
+            {{#if (gt item.LocalBindPort 0)}}
+              {{#let (concat (or item.LocalBindAddress '127.0.0.1') ':' item.LocalBindPort) as |combinedAddress| }}
+              <li class="port">
+                <CopyButtonFeedback 
+                  @copy={{combinedAddress}} 
+                  @name="Address to the clipboard" 
```

Comment:
Cool, I think its just this that needs making into "Address" otherwise the `title` will say "Copy Address to the clipboard to the clipboard"

```
// copy-buttonfeedback.hbs
(concat 'Copy ' name ' to the clipboard')
```

tbh I'm not even sure if that gets rendered anywhere, but if we did want to add 'to the clipboard' to the feedback tooltip that appears I'd say we should add that into the component itself, personally I wouldn't change that now tho.

Reply:
Ah, ok. Updating...